.changeset/funny-numbers-type.md

```diff
@@ -0,0 +1,6 @@
+---
+"remix": patch
+"@remix-run/dev": patch
+---
+
+update flat route module check for file colocation
```

Comment:
since flat route aren't released yet, do we really need a changeset?

Reply:
probably not, easy enough to clean up when changesets opens a PR though